src/components/containers/Fold.js

```diff
@@ -63,9 +66,26 @@ export default class Fold extends Component {
       </div>
     );
 
-    const foldContent = !folded ? (
-      <div className={contentClass}>{children}</div>
-    ) : null;
+    let foldContent = null;
+    if (!folded && !isEmpty) {
+      foldContent = <div className={contentClass}>{children}</div>;
+    }
+
+    if (!folded && isEmpty) {
+      foldContent = (
+        <div className={contentClass}>
+          <div className="fold__content__empty">
+            <div className="fold__content__empty__item">{icon}</div>
+            <div className="fold__content__empty__item">
+              {_('This trace does not yet have any data.')}
+            </div>
+            <div className="fold__content__empty__item">
+              {_('Return to the Graph > Create menu above to add data.')}
+            </div>
+          </div>
+        </div>
+      );
```

Comment:
Hmm, but what if a fold needs an empty state, but an icon is not passed?

```
 <div className="fold__content__empty__item">{icon ? icon : defaultIcon}</div>
```

I would maybe abstract this empty fold item out as something like 

```
<EmptyFoldContent
    show={!folded && isEmpty}
    icon={icon}
    message={['This trace does not yet have any data.', ''Return to the Graph > Create menu above to add data.']} />
```
What do you think @VeraZab ?

Reply:
Yeah maybe these messages: 'This trace does not yet have any data.' should not live in the `Fold` component, but be in the instance of the fold that needs the message?

Reply:
yeah you're right, I'll make a separate more generic component and I'll put a default icon